test/integration/asset_access_options_integration_test.rb

```diff
@@ -256,7 +256,7 @@ def setup_publishing_api_for(edition)
 
     def add_file_attachment_with_asset(filename, to:)
       to.attachments << FactoryBot.build(
-        :file_attachment_with_asset,
```

Comment:
My preferred fix would be to change the `asset_manager_id: "asset_manager_id_original"` to asset_manager_id: "asset_manager_id" which should hopefully allow us to replace `file_attachment_with_asset` with `file_attachment` - the generated URLs should be the same. Unless it, of course, breaks everything. 

I like the above better as it doesn't make me question why a csv was chosen for some tests. The csv usage activates my spidey-senses unnecessarily. And with this approach, csvs would only be used in the csv-relevant tests. But it doesn't really matter which doc type you use for the tests. I can live with this.

Reply:
I really hoped that we don't have to spend time looking too much into these tests as they are time sinks, but if we make these changes it might be needed. For example this tests is now making a csv file but we pass it a doc as filename (above in the code).